### PR TITLE
new-branch-v1

### PR DESCRIPTION
**Pull request by ConnorSmiley**
Sorry, I uninstall YARN but it screwed up my npm. So I had to start over

### CODE REVIEW

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -1,34 +1,39 @@
-import React from "react";
+import React from 'react';
+import '../styles/footer.css';
 
 const Footer = () => {
   return (
     <div className="footer">
-      <div className="col">
+      <div className="footer-col">
```
Comment (ConnorSmiley) [inline client/src/components/Footer.js:7]: I didn't chose the className="footer-col"

File: client/src/components/Header.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-				<Navbar />
-			</div>
-		</>
-	)
-}
-export default Header
+  return (
+    <>
+      <div className="tac">
+        <h1 className="header-title">Event Manager</h1>
```
Comment (ConnorSmiley) [inline client/src/components/Header.js:10]: changed from "header-header" => "header-title"

File: client/src/components/Search.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
 export default function Search() {
-	return (
-		<div>
-			<input type="text" placeholder="Search our site" />
-		</div>
-	)
+  return (
+    <div className="searchbar">
+      <input type="text" className="searchbar-input" placeholder="Search" />
```
Comment (ConnorSmiley) [inline client/src/components/Search.js:6]: Renamed from "searchbar--input" => "searchbar-input"

File: README.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ This service is meant to allow groups to manage events and members of their orga
   - ~~Facebook~~
   - ~~OnOffMix~~
 - Track posted events
+- ~~Track and report on event
```
Comment (TheBeege) [inline README.md:12]: Some bad formatting here

**Review by TheBeege — CHANGES_REQUESTED** [review]
Minor readme fix. Everything else looks okay, pending linting matches.

**Pull request closed without merging**